insert_data.sh

```diff
@@ -34,3 +34,47 @@ curl -X POST -H "Content-Type: application/json" \
 	}
 }' \
 http://localhost:9303/setup/data
+
+curl -X POST -H "Content-Type: application/json" \
```

Comment:
Do we need this request?

Reply:
I'm not sure, it depends on how the api handles it.

Reply:
Less of a question about the other posts (Although removing all the post requests does allow the tests to run) more about the one post request to 1234567890 as we are not using this mtditd in any scenarios

Reply:
Yeah just tested it, we don't need it